packages/react-union-scripts/scripts/webpack/loaders.parts.js

```diff
@@ -34,33 +12,30 @@ const loadBabel = include => () => ({
 	},
 });
 
-const loadScss = include => debug => ({
+const loadCss = include => (debug, ssr) => ({
```

Comment:
I don't think there's a reason to keep the loaders separate. This config can handle both extensions AFAIK.